WBooks/Development.xcconfig

```diff
@@ -0,0 +1,11 @@
+//
+//  Development.xcconfig
+//  WBooks
+//
+//  Created by Juan Silva on 03/06/2021.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+
```

Comment:
Both configuration files should have the same keys, otherwise the app would crash in "Release". So let's add "EXAMPLE = example_value" here too.